_episodes/interviews/2025-05-27-accesskit-with-matt-campbell-and-arnold-loubriat.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "AccessKit with Matt Campbell and Arnold Loubriat"
+date: 2025-05-27T20:00:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2025-05-29-accesskit-with-matt-campbell-and-arnold-loubriat.mp3
+length: "25041195"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+
+With [AccessKit](https://accesskit.dev/), [Matt Campbell](https://github.com/mwcampbell) and [Arnold Loubriat](https://github.com/DataTriny) [AccessKit](https://accesskit.dev/) took on the ambitious task of abstracting over
+the accessibility APIs of several target OS' to offer toolkit providers one unified way to make their UIs accessible across platforms.
```

Comment:
Looks like you've duplicated the link to AccessKit here (start and end of name list). Would also recommend mentioning that this is an interview of them, and that it's conducted by you :)

Reply:
Good catch. Thanks! 
Addressed this in with push.